### PR TITLE
Fix: Leave aliases to be unquoted in autocomplete

### PR DESCRIPTION
**Pull request by azmy60**
Testing CodeMirror outside of bks just little bit. It turns out that this behavior was not from the CodeMirror itself😅 .

Fixes point 2 in #1721

### CODE REVIEW

**Review by not-night-but — CHANGES_REQUESTED** [review]
@azmy60 Nice! Glad we got this one figured out. 

Code looks good, but do you think you could add some regression tests for this?

**Comment by azmy60** [comment]
Oh yeah, forgot that one. Let me see if I can do it.

File: apps/studio/src/lib/codemirror.ts [hunk 1]
```diff
@@ -1,13 +1,52 @@
 import CodeMirror from "codemirror";
 
-CodeMirror.defineMIME("text/x-pgsql", {
+export function autoquoteHandler(
+  instance: CodeMirror.Editor,
+  changeObj: CodeMirror.EditorChangeCancellable
+) {
+  if (!instance.getOption("mode").includes("pgsql")) return;
```
Comment (rathboma) [inline apps/studio/src/lib/codemirror.ts:7]: Quoting aliases isn't a problem in the other languages?

**Review by rathboma — COMMENTED** [review]
Looks good, question: is this a problem in other dialects?

**Comment by rathboma** [comment]
Ok for you to merge, but if also a problem in other dialects, they'll need sorting also. eg SQL Server has schemas + tables also

File: apps/studio/src/lib/codemirror.ts (hunk 1 above)
Comment (rathboma) [reply apps/studio/src/lib/codemirror.ts:7]: @azmy60 quick check-in for this question. If this isn't a problem, feel free to merge plz

**Review by rathboma — APPROVED** [review]
Feel free to merge this, but check my question first!

**Comment by azmy60** [comment]
Redshift needs the autoquote too so that's added. Merging now :)

**Comment by azmy60** [comment]
hmm.. I can't merge it somehow 🤔

**Pull request merged**